### ISSUE TITLE
Information being read from the wrong field during edit

### ISSUE THREAD

**Issue by jmaferreira** [keeps/roda#797]
To reproduce:

- Add info to "Name(s) of the creator(s)" and save the metadata
- Edit the metadata, and you'll see that it is also loaded on "Administrative and biographical history"

![screen shot 2017-01-31 at 14 04 12](https://cloud.githubusercontent.com/assets/3181302/22467913/5f04a540-e7be-11e6-81a8-2300dec2bf6e.png)

**Comment by hsilva-keep** [comment]
That is due to the wrong xpath for biohist (see roda-ui/roda-wui/src/main/resources/config/templates/ead_2002.xml.hbs):
```
{{~field name="bioghist"            order="180" type="text-area" labeli18n="crosswalks.dissemination.html.ead.bioghist" label="{\"en\": \"Administrative and biographical history\", \"pt\": \"Bibliografia\"}" xpath="/*:ead/*:archdesc/*:did/*:origination[@label='creator']/*:name/string()"~}}
{{~field name="creator"             order="190" labeli18n="crosswalks.dissemination.html.ead.creator" label="{\"en\": \"Name of creator(s)\", \"pt\": \"Nome do(s) autores\"}" xpath="/*:ead/*:archdesc/*:did/*:origination[@label='creator']/*:name/string()"~}}
```

**Comment by sleroux-keep** [comment]
Fixed @ f882b3f38e3a12d8bff50a345afd32b5e4ad96ac

**Comment by hsilva-keep** [comment]
I've just tested (42e06380cd5561ee695389c781cd7a1213febc28) and it's indeed fixed.
